app/routes/admin/reports/index.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  beforeModel() {
+    this._super(...arguments);
+    this.replaceWith('admin.reports.system-logs');
```

Comment:
@CosmicCoder96 @geekyd shouldn't we be using transitionTo instead of replaceWith here? 
Reference : https://guides.emberjs.com/v2.12.0/routing/redirection/